examples/extensions/remove_entire_sitelink_campaign_extension_setting.py

```diff
@@ -0,0 +1,230 @@
+            f'"{mutate_operation_responses[0].campaign_extension_setting_result.resource_name}".'
+        )
+        # Iterate through the remainder of the mutate operation responses,
+        # which are the extension feed item removal responses
+        for mutate_operation_response in mutate_operation_responses[1:]:
+            print(
+                "Removed an extension feed item with resource name "
+                f'"{mutate_operation_response.extension_feed_item_result}"'
+            )
+    except GoogleAdsException as ex:
+        print(
+            f'Request with ID "{ex.request_id}" failed with status '
+            f'"{ex.error.code().name}" and includes the following errors:'
+        )
+        for error in ex.failure.errors:
+            print(f'\tError with message "{error.message}".')
+            if error.location:
+                for field_path_element in error.location.field_path_elements:
+                    print(f"\t\tOn field: {field_path_element.field_name}")
+        sys.exit(1)
+
+
+def _create_sitelink_campaign_extension_setting_mutate_operation(
+    client, customer_id, campaign_id
+):
+    """Creates a MutateOperation for the sitelink campaign extension setting
+    that will be removed.
+
+    Args:
+        client: an initialized GoogleAdsClient instance
+        customer_id: the client customer ID.
+        campaign_id: the campaign ID.
+
+    Returns:
+        The created MutateOperation for the sitelink campaign extension
+            setting.
+    """
+    extension_type_enum = client.get_type("ExtensionTypeEnum", version="v6")
+    composite_id = ResourceName.format_composite(
+        campaign_id,
+        extension_type_enum.ExtensionType.Name(
+            extension_type_enum.ExtensionType.SITELINK
+        ),
+    )
+    # Construct the campaign extension setting resource name, in format:
+    # customers/{customer_id}/campaignExtensionSettings/{campaign_id}~{extension_type}
+    resource_name = client.get_service(
+        "CampaignExtensionSettingService", version="v6"
+    ).campaign_extension_setting_path(customer_id, composite_id)
+
+    # Create a MutateOperation for the campaign extension setting.
+    mutate_operation = client.get_type("MutateOperation", version="v6")
+    mutate_operation.campaign_extension_setting_operation.remove = resource_name
+    return mutate_operation
+
+
+def _get_all_sitelink_extension_feed_items(
+    client, ga_service, customer_id, campaign_id
+):
+    """Gets all sitelink extension feed items associated to the specified
+    campaign extension setting.
+
+    Args:
+        client: an initialized GoogleAdsClient instance.
+        ga_service: the Google Ads API service client.
+        customer_id: the client customer ID.
+        campaign_id: the ID of the campaign to get the sitelink extension feed
+            items from.
+
+    Returns:
+        An array of str resource names of extension feed items.
```

Comment:
should mention Raises in the PyDoc per https://google.github.io/styleguide/pyguide.html#244-decision

Reply:
I will do this if I don't end up going with printing and exiting